tbtc-ts/src/bitcoin.ts

```diff
@@ -83,6 +90,73 @@ export type UnspentTransactionOutput = TransactionOutpoint & {
   value: number
 }
 
+/**
+ * Represents data of a decomposed transaction.
+ */
+export interface DecomposedRawTransaction {
+  /**
+   * Transaction's version.
+   */
+  version: string
+
+  /**
+   * All transaction's inputs, prepended by the number of transaction inputs.
+   */
+  inputs: string
+
+  /**
+   * All transaction's outputs prepended by the number of transaction outputs.
```

Comment:
`All transaction's outputs prepended by the number of transaction outputs, as an un-prefixed hex string.`

Reply:
Done.